packages/arb-token-bridge-ui/src/hooks/useRedeemRetryable.ts

```diff
@@ -39,11 +41,15 @@ export function useRedeemRetryable(): UseRedeemRetryableResult {
     try {
       setIsRedeeming(true)
 
+      if (!signer) {
+        throw 'Signer is undefined'
```

Comment:
When would this error occur? If we know a root cause we can probably communicate that to user in the throw error for better troubleshooting.

Reply:
1) when the rpc hasn't gotten back with the info yet, which is a very short period, almost negligible as it'll be updated once it's fetched
2) https://github.com/OffchainLabs/arbitrum-token-bridge/pull/651#discussion_r1169751448